pages/index.vue

```diff
@@ -135,31 +183,79 @@ export default {
 
       return notesInScale;
     },
-    drawScale() {
-      this.resetStrings();
+    getNotesInCurrentChord() {
+      const currentToneIndex = tones.indexOf(this.currentTone);
+      const tonesStartingInSelected = tones.map((_, i) => tones[(currentToneIndex + i) % tones.length]);
 
-      const notesInScale = this.getNotesInCurrentScale();
+      const notesInChord = [tonesStartingInSelected[0]];
+      const selectedChord = chords[this.currentChord];
+      let cursor = 0;
+      for (let i = 0; i < selectedChord.length; i ++) {
+        cursor = (cursor + selectedChord[i]) % tonesStartingInSelected.length;
+        notesInChord.push(tonesStartingInSelected[cursor]);
+      }
```

Comment:
Maybe this function and `getNotesInCurrentScale` could be abstracted into a single function.